nirc_ehr/resources/queries/study/protocolAssignment.js

```diff
@@ -29,7 +29,7 @@ function getLastAssignment(id){
 
 function onInit(event, helper){
     helper.setScriptOptions({
-        allowAnyId: true,
+        allowAnyId: false,
```

Comment:
Please test protocol and project allowAnyId again when you get the other PR combining protocol/project with birth/arrival merged in.